extension/chrome/settings/modules/change_passphrase.ts

```diff
@@ -108,6 +108,7 @@ View.run(class ChangePassPhraseView extends View {
     }
     await KeyStore.add(this.acctEmail, this.primaryPrv!);
     const persistentlyStoredPp = await PassphraseStore.get(this.acctEmail, this.primaryKi!.fingerprints[0], true);
+    // todo:
```

Comment:
should we leave this copying as it is or rely on OrgRules?
Is there a reason to save `undefined` to local storage (line 112) in case it was read as `undefined`  on line 110?

Reply:
same as https://github.com/FlowCrypt/flowcrypt-browser/pull/3707/files#r646027379 - good to add sanity checks here with errors thrown. 

Undefined - no reason / no difference (see other comment). Just fewer lines of code to write :o)